### DISCUSSION TITLE
SSL certificate not generated for www subdomain

### DISCUSSION THREAD

**Discussion by Sk92** [vercel/community#4853] [Help] [answered]
### Summary

We are running into an issue where our main domain (247x.io) has a valid SSL certificate, but the www. subdomain has an old one (expired).

It seems that Vercel didn't generate correctly the SSL certificate after we connected the subdomain.
The www subdomain should be configured properly (CNAME to cname.vercel-dns.com).

What is happening?

### Example

https://www.247x.io

### Steps to Reproduce

1. Create a deploy
2. Connect the root domain
3. After some days, connect the subdomain too

-> The subdomain won't have a valid SSL certificate

**Comment by swarnava** [comment, marked as answer]
Hi,

It looks like you have only added 247x.io in your project. Can you also add www.247x.io and give it a try?

**Comment by Sk92** [comment]
Worked! 
I thought Vercel automatically generate a wildcard certificate. 
Thanks 🙏
